index.html

```diff
@@ -181,6 +181,66 @@ <h2>Requests/responses sharing and proxy between Native and WebView</h2>
         </dl>
       </section>
 	    
+      <section>
+        <h2>The Origin in a WebView for locally hosted content</h2>
+        <dl>
+            <dt>Submitter(s)</dt>
+            <dd>
+            Niklas Merz, Apache Software Foundation
+            </dd>
+            
+            <dt>Motivation</dt>
+            <dd>
+            <a href="https://appfigures.com/top-sdks/development/all">Up to 30%</a> of apps found in the App stores (Apple and Google) are built with frameworks like <a href="https://cordova.apache.org">Apache Cordova</a> and <a href="https://capacitorjs.com/">CapacitorJS</a>. Those two frameworks use one big Webview for providing app developers a native wrapper and some plugins for their Web App. App developers build their Web application and put the HTML, CSS and JavaScript files in one folder. The framework then takes care of building a Native App project and bundling the Web code as a Native application ready to distribute via the App stores.
+            <br>
+            App developers usually only need knowledge in HTML, CSS and JavaScript and can call native OS features via plugins. The frameworks provide a bridge that makes these native features available as JavaScript functions inside the Webview. Because of this, the locally hosted Web application "should just work as published on the Web". Connections to the backend are usually done via fetch/XHR. Because the origin for the App code is different than the backend, there is always CORS involved.
```

Comment:
```suggestion
            App developers usually only need knowledge in HTML, CSS and JavaScript and can call native OS features via plugins. The frameworks provide a bridge that makes these native features available as JavaScript functions inside the WebView. Because of this, the locally hosted Web application "should just work as published on the Web". Connections to the backend are usually done via fetch/XHR. Because the origin for the app code is different than the backend, there is always CORS involved.
```